### PR TITLE
Task add support for gpu device in docker

### PR DESCRIPTION
**Pull request by jasiu86**
GPU RUN:
https://ci.iotg.sclab.intel.com/job/ovmsc/job/OVMSCOps/5362/

### CODE REVIEW

File: tests/functional/config.py [hunk 1]
```diff
@@ -66,6 +63,13 @@
 """ TT_TARGET_DEVICE - one of "CPU", "GPU" """
 target_device = os.environ.get("TT_TARGET_DEVICE", "CPU")
 
+"""IMAGE - docker image name which should be used to run tests"""
+if target_device == "CPU":
```
Comment (pgierasi) [inline tests/functional/config.py:67]: in case of HDDL & NCS we would like to use the default Image (non-GPU)

**Review by pgierasi — CHANGES_REQUESTED** [review]
In case of HDDL & NCS we would like to use the default Image (non-GPU)

File: tests/functional/config.py (hunk 1 above)
Comment (jasiu86) [reply tests/functional/config.py]: Done

**Comment by jasiu86** [comment]
> In case of HDDL & NCS we would like to use the default Image (non-GPU)

Fixed

File: tests/functional/config.py [hunk 2]
```diff
@@ -78,3 +82,15 @@
...
+""" TT_DEFAULT_INFER_TIMEOUT - Timeout for CPU target device"""
+default_infer_timeout = get_int("TT_DEFAULT_INFER_TIMEOUT", 10)
+
+""" TT_DEFAULT_GPU_INFER_TIMEOUT - Timeout for CPU target device"""
+default_gpu_infer_timeout = get_int("TT_DEFAULT_GPU_INFER_TIMEOUT", 10*default_infer_timeout)
+
+""" INFER TIMEOUT """
+if target_device == "GPU":
+    infer_timeout = default_gpu_infer_timeout
+else:
```
Comment (pgierasi) [inline tests/functional/config.py:95]: NCS and HDDL devices are slower than CPU

**Review by pgierasi — APPROVED** [review]

File: tests/functional/config.py (hunk 2 above)
Comment (jasiu86) [reply tests/functional/config.py:95]: NCS and HDDL will be changed in following tasks and PR.

**Review by krzyczar — APPROVED** [review]

**Pull request merged**